### PR TITLE
disable version checks on CI

### PR DESCRIPTION
**Pull request by matklad**
Is anyone actively looking into fixing this properly? I might do that (look, not necessary fix)!

### CODE REVIEW

File: .github/workflows/clients-node-build.yml [hunk 1]
```diff
@@ -34,12 +34,15 @@ jobs:
...
-  version_check:
-    needs: [shellcheck_scripts, test_distro_install]
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - run: npm install && ./scripts/version_check.sh
+# Disable version check for now, as it basically requires us to bump version in package.json for every chage.
+# Now that bors enforces that our CI is green, that becomes very painful.
+#
+#  version_check:
```
Comment (sentientwaffle) [inline .github/workflows/clients-node-build.yml:42]: I think this job needs to be removed from `publish`'s dependencies.

Comment (matklad) [reply .github/workflows/clients-node-build.yml:42]: Good catch!

**Review by sentientwaffle — COMMENTED** [review]
bors r+

Eventually we should fix this (or replace it with something similar, e.g. a scheduled release) but for now it is too noisy. For now, changes which require a new client version should increment the `package.json`'s version manually.

**Review by sentientwaffle — DISMISSED** [review]

**Comment by matklad** [comment]
BTW, I think if you re r+ https://github.com/tigerbeetledb/tigerbeetle/pull/387, it should be correctly pipelined after this PR, even without waiting for this to get to master (not 100% sure though)

**Comment by matklad** [comment]
Heh, ironically, as soon as I mentioned on slack that bors obviates the need to rebase PRs on top of master, I exactly hit the problem due to not rebasing on master (this was based on pre-bors branch, so I didn't remove version check from bors checks, which got the CI stuck)

**Review by sentientwaffle — APPROVED** [review]
bors r+

**Review by sentientwaffle — APPROVED** [review]
bors r+

**Pull request merged**